Tests/Integration/PauseWithPredisTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Resque\Plugins\Tests\Integration;
+
+use Resque;
+use Predis\Client;
+
+/**
+ * Pause tests.
+ *
+ * @package     PHP Resque Pause
+ * @author      Wedy Chainy <wedy.chainy@bigcommerce.com>
+ * @license     http://www.opensource.org/licenses/mit-license.php
+ */
+class PauseWithPredisTest extends PauseTestBase
+{
+    public static function setUpBeforeClass()
+    {
+        $testMisc = realpath(__DIR__ . '/misc/');
+        $redisConf = "$testMisc/redis.conf";
+
+        // Attempt to start our own redis instance for tesitng.
+        exec('which redis-server', $output, $returnVar);
+        if ($returnVar != 0) {
+            echo "Cannot find redis-server in path. Please make sure redis is installed.\n";
+            exit(1);
+        }
+
+        exec("cd $testMisc; redis-server $redisConf", $output, $returnVar);
+        usleep(500000);
```

Comment:
This code was borrowed from https://github.com/chrisboulton/php-resque/blob/master/test/bootstrap.php#L25
I think I'm going to strip all of this out. If it can't connect to redis it should say to start it and that should be part of the travis.yml